blocks/api/raw-handling/test/inline-block-corrector.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * External dependencies
+ */
+import { equal } from 'assert';
```

Comment:
Curious why we're using the built-in `assert` lib here rather than Jest?

Reply:
It's been used all over this paste since the beginning. I can change this.

Reply:
Not a fan of the globals :|